pl_examples/domain_templates/computer_vision_fine_tuning.py

```diff
@@ -276,10 +274,23 @@ def add_model_specific_args(parent_parser):
             help="Name (as in ``torchvision.models``) of the feature extractor",
         )
         parser.add_argument(
-            "--epochs", default=15, type=int, metavar="N", help="total number of epochs", dest="nb_epochs"
+            "--epochs", default=5, type=int, metavar="N", help="total number of epochs", dest="nb_epochs"
+        )
+        parser.add_argument(
+            "--limit_train_batches",
+            default=1.0,
+            type=float,
+            help="How much of training dataset to check (floats = percent, int = num_batches)"
+        )
+        parser.add_argument(
+            "--limit_val_batches",
+            default=1.0,
+            type=float,
+            help="How much of validation dataset to check (floats = percent, int = num_batches)"
```

Comment:
the trainer already adds these arguments, there is a collision now as they cannot exist twice

Reply:
The trainer isn't used with the parser.